File: 6TH_SEMESTER/SWE 4604 - SOFTWARE TESTING AND QUALITY ASSURANCE LAB/6/testinglab06/src/tests/LoginPage.java
package tests;

import org.junit.jupiter.api.Test;
import org.openqa.selenium.By;
import org.openqa.selenium.WebElement;

//210042112

public class LoginPage extends BaseTest {
    @Test
    public void TestLogIn()
    {
        driver.get("https://practicesoftwaretesting.com/auth/login");

        WebElement email = driver.findElement(By.id("email"));
        email.sendKeys("customer@practicesoftwaretesting.com");//taken from the pdf provided

        WebElement password = driver.findElement(By.id("password"));
        password.sendKeys("welcome01");//taken from the pdf provided

        WebElement loginbtn = driver.findElement(By.className("btnSubmit"));
        loginbtn.click();

    }

}
// -_- N4M154 -_-

File: 6TH_SEMESTER/SWE 4604 - SOFTWARE TESTING AND QUALITY ASSURANCE LAB/6/Lab06CSharp/TestProject1/LogInPage.cs
using OpenQA.Selenium;

namespace Lab06
{
    public class LogInPage : BaseTest
    {
        [Fact]
        public void TestLogIn()
        {
            driver.Navigate().GoToUrl("https://practicesoftwaretesting.com/auth/login");

            var email = driver.FindElement(By.Id("email"));
            email.SendKeys("customer@practicesoftwaretesting.com");//taken from the pdf provided

            var password = driver.FindElement(By.Id("password"));
            password.SendKeys("welcome01");//taken from the pdf provided

            var loginbtn = driver.FindElement(By.ClassName("btnSubmit"));
            loginbtn.Click();

        }
    }
}
